code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -212,9 +212,13 @@
 ////////////////////////////////////////////
 
 //Returns a list of damaged organs
-/mob/living/carbon/human/proc/get_damaged_organs(var/brute, var/burn)
+/mob/living/carbon/human/proc/get_damaged_organs(var/brute, var/burn, var/flag = "BOTH") // flag takes ORG, SYNTH, OR BOTH for consistancy

```

Comment:
Generally for things like this, we'll use defines rather than strings. So you could, say use `AFFECT_ROBOTIC_ORGAN`, `AFFECT_ORGANIC_ORGAN` and `AFFECT_ALL_ORGAN` as defines and plug them in instead of strings. Functionally there's not really a difference, its mostly just code consistency.